overlord/configstate/configcore/homedirs.go

```diff
@@ -88,6 +82,10 @@ func configureHomedirsInAppArmorAndReload(homedirs []string, opts *fsOnlyContext
 
 	// Only reload profiles if it's during runtime
```

Comment:
this needs updating, // Only update apparmor configuration snippet and reload profiles if ...

Reply:
Updated!